dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java

```diff
@@ -130,8 +131,14 @@ public LogTraceData parse(RoutedMessage routedMessage, LogRecord logRecord) {
         logData.setModule(logRecord.getLoggerName());
         logData.setSeverity(LogFormatUtils.mapLevelToType(logRecord));
         logData.setLoglevel(LogFormatUtils.mapLevelToRawType(logRecord));
-        logData.setMethodName(logRecord.getSourceMethodName());
-        logData.setClassName(logRecord.getSourceClassName());
+
+        if (logRecord.getLoggerName() != null & logRecord.getLoggerName().equals(SYSOUT)) {
+            logData.setMethodName("");
```

Comment:
should this not also watch for SystemErr?

Reply:
Added SystemErr handling